src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -530,4 +551,22 @@ private function checkProductQuantity(
             }
         }
     }
+
+    private function compareItemSubtotal(ProductInterface $product, \Closure $price): void
+    {
+        /** @var ChannelPricingInterface $pricing */
+        $pricing = $product->getVariants()->get(0)->getChannelPricingForChannel($this->sharedStorage->get('channel'));
+
+        $items = $this->responseChecker->getValue($this->cartsClient->getLastResponse(), 'items');
+
+        foreach ($items as $item) {
```

Comment:
```suggestion
        $productName = $product->getName();
        foreach ($items as $item) {
        if ($item['productName'] === $productName()) {
```
in this `$product->getName()` we will call once but I assume there is not a lot of items in the $items.

Reply:
You wrong, look in this scenario: https://github.com/Sylius/Sylius/pull/12081/files#diff-bb3427020ccf457607272ab922033ec2e77d77d2226fcd3e6e9089647b77a7b8L8-L9